tests/integration_test/integration_test.sh

```diff
@@ -44,408 +44,573 @@
+print_style "info" "USE_HMAC=${USE_HMAC}"
+print_style "info" "SUBMIT_WDL=${SUBMIT_WDL}"
+print_style "info" "MAX_CROMWELL_RETRIES=${MAX_CROMWELL_RETRIES}"
+print_style "info" "TENX_ANALYSIS_WDLS=${TENX_ANALYSIS_WDLS}"
+print_style "info" "TENX_OPTIONS_LINK=${TENX_OPTIONS_LINK}"
+print_style "info" "TENX_SUBSCRIPTION_ID=${TENX_SUBSCRIPTION_ID}"
+print_style "info" "TENX_WDL_STATIC_INPUTS_LINK=${TENX_WDL_STATIC_INPUTS_LINK}"
+print_style "info" "TENX_WDL_LINK=${TENX_WDL_LINK}"
+print_style "info" "TENX_WORKFLOW_NAME=${TENX_WORKFLOW_NAME}"
+print_style "info" "TENX_VERSION=${TENX_VERSION}"
+print_style "info" "SS2_ANALYSIS_WDLS=${SS2_ANALYSIS_WDLS}"
+print_style "info" "SS2_OPTIONS_LINK=${SS2_OPTIONS_LINK}"
+print_style "info" "SS2_SUBSCRIPTION_ID=${SS2_SUBSCRIPTION_ID}"
+print_style "info" "SS2_WDL_STATIC_INPUTS_LINK=${SS2_WDL_STATIC_INPUTS_LINK}"
+print_style "info" "SS2_WDL_LINK=${SS2_WDL_LINK}"
+print_style "info" "SS2_WORKFLOW_NAME=${SS2_WORKFLOW_NAME}"
+print_style "info" "SS2_VERSION=${SS2_VERSION}"
+print_style "info" "VAULT_TOKEN_PATH=${VAULT_TOKEN_PATH}"
+print_style "info" "PWD=${PWD}"
+print_style "info" "LIRA_IMAGE_VERSION=${LIRA_IMAGE_VERSION}"
 
+docker run -i --rm \
+              -e LIRA_ENVIRONMENT="${LIRA_ENVIRONMENT}" \
+              -e CROMWELL_URL="${CROMWELL_URL}" \
+              -e USE_CAAS="${USE_CAAS}" \
+              -e COLLECTION_NAME="${COLLECTION_NAME}" \
+              -e GCLOUD_PROJECT="${GCLOUD_PROJECT}" \
+              -e GCS_ROOT="${GCS_ROOT}" \
+              -e LIRA_VERSION="${LIRA_VERSION}" \
+              -e DSS_URL="${DSS_URL}" \
+              -e SCHEMA_URL="${SCHEMA_URL}" \
+              -e INGEST_URL="${INGEST_URL}" \
+              -e USE_HMAC="${USE_HMAC}" \
+              -e SUBMIT_WDL="${SUBMIT_WDL}" \
+              -e MAX_CROMWELL_RETRIES="${MAX_CROMWELL_RETRIES}" \
+              -e TENX_ANALYSIS_WDLS="${TENX_ANALYSIS_WDLS}" \
+              -e TENX_OPTIONS_LINK="${TENX_OPTIONS_LINK}" \
+              -e TENX_SUBSCRIPTION_ID="${TENX_SUBSCRIPTION_ID}" \
+              -e TENX_WDL_STATIC_INPUTS_LINK="${TENX_WDL_STATIC_INPUTS_LINK}" \
+              -e TENX_WDL_LINK="${TENX_WDL_LINK}" \
+              -e TENX_WORKFLOW_NAME="${TENX_WORKFLOW_NAME}" \
+              -e TENX_VERSION="${TENX_VERSION}" \
+              -e SS2_ANALYSIS_WDLS="${SS2_ANALYSIS_WDLS}" \
+              -e SS2_OPTIONS_LINK="${SS2_OPTIONS_LINK}" \
+              -e SS2_SUBSCRIPTION_ID="${SS2_SUBSCRIPTION_ID}" \
+              -e SS2_WDL_STATIC_INPUTS_LINK="${SS2_WDL_STATIC_INPUTS_LINK}" \
+              -e SS2_WDL_LINK="${SS2_WDL_LINK}" \
+              -e SS2_WORKFLOW_NAME="${SS2_WORKFLOW_NAME}" \
+              -e SS2_VERSION="${SS2_VERSION}" \
+              -v "${VAULT_TOKEN_PATH}":/root/.vault-token \
+              -v "${PWD}/lira/kubernetes":/working \
+              broadinstitute/dsde-toolbox:ra_rendering \
+              /usr/local/bin/render-ctmpls.sh \
+              -k "/working/${LIRA_CONFIG_FILE}.ctmpl" || true
```

Comment:
What is the `|| true` for?

Reply:
Something was going on in this script where it rendered the templates and exited the script. No errors, and it completed the script that it called in the docker image, but it just exited. The || true means that it evaluates the rendering script and either way returns to this method. If the files aren't rendered then they will break later in the script (within the next few lines of the test).

Reply:
Ok, thanks for the explanation!